src/classes/LocationResponse.ts

```diff
@@ -1,8 +1,6 @@
 import UseCaseResponse from './UseCaseResponse';
 
-class LocationResponse extends UseCaseResponse {
-  latitude: number;
-
-  longitude: number;
+class LocationResponse implements UseCaseResponse {
+  constructor(readonly latitude: number, readonly longitude: number) {}
```

Comment:
That is working? Du setzt Attribute die die Klasse gar nicht enthält. Zumindestens für die Readability würde ich die Attribute behalten.

Reply:
Ja, das nennt sich parameter properties: https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties
Ich finde das eigentlich ganz elegant so, hält das ganze recht kurz und ist imho nicht weniger übersichtlich wenn man die Syntax nennt.
Ich hab den Konstruktor eingebaut, damit man die Attribute nicht in einem Extraschritt setzen muss.

Reply:
nichts gegen den Konstruktor,  ich finde es macht es halt schwerer direkt zu erkennen was sind das für Parameter.
Wen ich jetzt nen use Case mache will ich kurz in das file reinsehen und sehe welche Attribute ich dort brauche.

Reply:
Ich glaube, bei so simplen Klassen, die nur ein oder zwei Attribute haben, ist der Konstruktor relativ selbsterklärend. Und so kann man halt keine "falschen" Objekte erstellen, bei denen ein Attribut nicht gesetzt wurde.
Und deine IDE sagt dir hoffentlich, welche Attribute du brauchst 😁

Reply:
Mach Atom im Raw nicht, nein